src/test/java/codeu/model/data/ModelDataTestHelpers.java

```diff
@@ -237,4 +247,35 @@ public User build() {
       return user;
     }
   }
+
+  /**
+   * Use this to create a fake Hashtag to use in a unit test. When created it contains random data
+   * in every field, and the individual methods can be used to set the test conditions. For example,
+   * if the test needs specific owner ID and title, then you could do:
+   *
+   * <pre>
+   * {
+   * 	&#64;code
+   * 	UUID fakeOwner = UUID.randomUUID();
+   * 	String fakeContent = "test content 1";
+   * 	Hashtag fakeHashtag = new TestHashtagBuilder().withOwnerId(fakeOwner).withContent(fakeContent).build();
+   * }
+   * </pre>
+   */
+  public static class TestHashtagBuilder {
```

Comment:
For this class, you need to follow the same format as the above code by adding functions for all the fields such as `public TestHashtagBuilder withUserSource(String userSource) {`